README.md

```diff
@@ -60,8 +60,8 @@ Manual install:
 
 1) Download a .tgz file based on your Operating System / Arch
 2) Unpack the assembly
-3) move `chectl` folder into a folder like `$HOME/chectl`
-4) add `$HOME/chectl/bin` to `$PATH``
+3) Move `chectl` folder into a folder like `$HOME/chectl`
+4) Use `$HOME/chectl/bin/run` to run chectl commands
```

Comment:
I would add an optional step to add `chectl` alias

Reply:
Could you provide the command?

Reply:
It depends on the user environment